common/usb_cdc.c

```diff
@@ -183,11 +183,12 @@ static const char StrDescManufacturer[] = {
 };
 
 static const char StrDescProduct[] = {
-  4,			// Length
+  9,			// Length

```

Comment:
Hmm. There are docs that say that length must be even and that the strings are not nul byte terminated. Hence 8 instead of 4 would be a correct fix.

Reply:
BTW: the wrong length of 4 should only result in a truncated string. This is for human eyes only and doesn't have a function in USB communication.

Reply:
Ok. Do you have any suggestions? Having the length be 8 causes the device to not be accessible using the Android USB Host API. adding a space at the end of the string? making it the full "proxmark3" rather than "PM3"?